src/components/Showcase/Highlight/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { Theme } from '@theme'
+import { Link } from 'gatsby'
+
+import * as S from './styled'
+import { getProjectTheme } from '../../../utils/getProjectTheme'
+import { IProject } from '../../../pages/showcase'
+import * as T from '../../UI/Typography'
+
+interface IHighlight {
+  project: IProject
+}
+
+const Highlight: React.FC<IHighlight> = ({ project }) => {
+  const {
+    services,
+    title,
+    tagline,
+    seo,
+    fields,
+    highlightTextColumnOn = 'left',
+  } = project.node
+  const contentOnLeft = highlightTextColumnOn === 'left'
+
+  const imageElement = (
+    <S.Image fluid={project.node.thumbnail.childImageSharp.fluid} />
+  )
+  const contentElement = (
+    <S.Content>
+      <div>
```

Comment:
why this div?

Reply:
To center vertically

Reply:
why don't you use <S.Content> ?

Reply:
well, I use it, but I still need a child element to handle the content:
```jsx
export const Content = styled.div`
  display: flex;

  > div {
    margin: auto;
  }
```

Right?

Reply:
If you need an element there, please name it accordingly so the next person that picks up the code doesn't fuck up your logic.

i.e.: next person comes, sees that div and goes "oh, this would be more semantically correct with an <article> or something." and changes it. your code breaks.

Rename that div too S.InnerContent or something that conveys the message that it does have some style associated with it